src/components/Sum.jsx

```diff
@@ -1 +1,20 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Sum({ a, b }) {
```

Comment:
Also, you can use arrow function expression for functions, where your intermediately return a value